scripts/validate_docstrings.py

```diff
@@ -491,12 +492,14 @@ def first_line_ends_in_dot(self):
         if self.doc:
             return self.doc.split('\n')[0][-1] == '.'
 
+    @property
+    def deprecated_with_directive(self):
+        return ('.. deprecated:: ' in (self.summary + self.extended_summary))
```

Comment:
```suggestion
        return '.. deprecated:: ' in (self.summary + self.extended_summary)
```

unnecessary parenthesis

Reply:
Updated,